coderd/database/queries/workspaceagentstats.sql

```diff
@@ -341,11 +343,15 @@ WITH agent_stats AS (
 )
 SELECT
 	users.username, workspace_agents.name AS agent_name, workspaces.name AS workspace_name, rx_bytes, tx_bytes,
-	session_count_vscode, session_count_ssh, session_count_jetbrains, session_count_reconnecting_pty,
-	connection_count, connection_median_latency_ms
+	coalesce(session_count_vscode, 0)::bigint AS session_count_vscode,
+	coalesce(session_count_ssh, 0)::bigint AS session_count_ssh,
+	coalesce(session_count_jetbrains, 0)::bigint AS session_count_jetbrains,
+	coalesce(session_count_reconnecting_pty, 0)::bigint AS session_count_reconnecting_pty,
+	coalesce(connection_count, 0)::bigint AS connection_count,
+	connection_median_latency_ms
 FROM
 	agent_stats
-JOIN
+LEFT JOIN
```

Comment:
It's possible for the subquery (`latest_agent_stats`) that selects the rows with session counts from the last minute to be empty, if there are no sessions. This needs to be a left join if we want to update the prometheus metrics (such as going from n sessions to 0), or the rx/tx bytes.